webview-ui/src/components/chat/TaskHeader.tsx

```diff
@@ -141,7 +141,9 @@ const TaskHeader = ({
 								{highlightText(task.text, false, customModes)}
 							</div>
 						</div>
-						{task.images && task.images.length > 0 && <Thumbnails images={task.images} />}
+						{task.images && task.images.length > 0 && task.files && task.files.length > 0 && (
```

Comment:
Shouldn't this be something like:

```suggestion
						{(task.images && task.images.length > 0 ||task.files && task.files.length > 0) && (
```

i.e. images or files instead of images and files?